app/schema.graphql

```diff
@@ -1472,6 +1478,18 @@ type PromptVersionEdge {
   node: PromptVersion!
 }
 
+input PromptVersionInput {
+  description: String = null
+  templateType: PromptTemplateType!
+  templateFormat: PromptTemplateFormat!
+  template: JSON!
+  invocationParameters: JSON = null
+  tools: JSON = null
+  outputSchema: JSON = null
+  modelName: String!
+  modelProvider: String!
```

Comment:
I don't think we can have these required? I guess it's good to start required...